shuffle/move/examples/Message/Move.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "Message"
+version = "0.0.0"
+
+[addresses]
+MessageAddress = "0x2"
+
+[dependencies]
+MoveStdlib = { local = "../../../../language/move-stdlib", addr_subst = { "Std" = "0x1" }}
```

Comment:
Are we planning to have the stdlib inside the tool somehow? It seems odd we need to specify this.

Reply:
We are, but I believe due to some limitations with the Move package system, we're temporarily referencing it in the Move.toml. This unblocks us from forward progress, but ultimately I think this work item belongs to Move packaging.